randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -985,6 +985,11 @@ Extra - room_id: [27]
                       Wall Jump (Intermediate)
               # NHJ with ledge grabs: https://youtu.be/YdMD2TAgkuc
               Movement (Intermediate)
+          All of the following:
+              # Break the block with speed booster
+              Speed Booster
+              # NHJ With ledge grabs: https://youtu.be/V20W0F4dZP0
```

Comment:
This seems like a redundant block since this method it already documented on line 986. (also wrong video)

Reply:
Oh, I didn't realize this was already documented. I can remove this